R/pool.R

```diff
@@ -186,7 +259,11 @@ Pool <- R6Class("Pool",
           NULL
         )
         if (!is.null(removeFrom)) {
-          rm(list = id, envir = removeFrom)
```

Comment:
In fact you could pretty much do `changeObjectStatus = function(object, to)` at this point, everything else is redundant. I'm not sure we should do that before merging to master though, since it will change a bunch more lines of code.